lib/screens/home/cocktail_menu.dart

```diff
@@ -1,9 +1,10 @@
 enum CocktailMenu {
-  categories(name: 'Category'),
-  typeOfGlass(name: 'Type of Glass'),
-  ingredient(name: 'Ingredient'),
-  alcoholic(name: 'Alcoholic/Non Alcoholic');
+  categories(name: 'Category', path: 'categories'),
+  typeOfGlass(name: 'Type of Glass', path: 'typeOfGlass'),
+  ingredient(name: 'Ingredient', path: 'ingredients'),
+  alcoholic(name: 'Alcoholic/Non Alcoholic', path: 'alcoholicNonAlcoholic');
```

Comment:
- [ ] Let's add all this name in a Localizable Strings, you will probably need to move the name into an extension of `CocktailMenu`  because it needs to be  a constant in order to be in the Enum constructor.